src/logic/handlers/textures-handler.ts

```diff
@@ -64,10 +64,12 @@ export const createTexturesEventHandler = ({
           await queuesStatusManager.markAsFinished(event.metadata.platform, event.metadata.entityId)
         }
 
-        const status: Registry.SimplifiedStatus = await entityStatusFetcher.fetchBundleStatus(
-          event.metadata.entityId,
-          event.metadata.platform
-        )
+        const status: Registry.SimplifiedStatus =
+          event.metadata.statusCode === ManifestStatusCode.SUCCESS ||
+          event.metadata.statusCode === ManifestStatusCode.CONVERSION_ERRORS_TOLERATED ||
+          event.metadata.statusCode === ManifestStatusCode.ALREADY_CONVERTED
+            ? Registry.SimplifiedStatus.COMPLETE
+            : Registry.SimplifiedStatus.FAILED
```

Comment:
Could it be possible to have a mismatch between the db status and the one that comes in the metadata? Would that be a problem?